app/app_test.go

```diff
@@ -0,0 +1,29 @@
+package app_test
+
+import (
+	"testing"
+
+	"gorm.io/gorm"
+
+	mopsos "github.com/adfinis-sygroup/mopsos/app"
+)
+
+func Test_NewAppFailWhenNoDatabase(t *testing.T) {
+	_, err := mopsos.NewApp(&mopsos.Config{
+		HttpListener:  ":8080",
+		EnableTracing: false,
+		TracingTarget: ""}, nil)
+	if err == nil {
+		t.Errorf("error: %v", err)
```

Comment:
```suggestion
		t.Error("expected error, got nil")
```

i should also check for specific errors and also include a check for missing config/config-validation